src/chart/cartesianChart.js

```diff
@@ -0,0 +1,232 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.chart.cartesianChart = function(xScale, yScale) {
+
+        xScale = xScale || d3.scale.linear();
+        yScale = yScale || d3.scale.linear();
+
+        var xAxisHeight = 20,
+            yAxisWidth = 40,
+            yAxisLabelWidth = 20,
+            xAxisLabelHeight = 20,
+            titleHeight = 20,
+            yAxisLabel = 'y-axis',
+            xAxisLabel = 'x-axis',
+            title = 'Chart Title',
```

Comment:
Good idea, I've also renamed `titleHeight` to `chartLabelHeight` for consistency.